assets/src/components/form/select/model-selects/status.js

```diff
@@ -97,6 +97,7 @@ export default class StatusSelect extends Component {
 			...this.props,
 			...selectOpts,
 			...this.state,
+			optionsEntityMap,
```

Comment:
why was this added?  It should be covered fine by the `...this.props` spread (it would have been filled by `defaultProps`)?

Reply:
missed that it was added in by `defaultProps`